package.json

```diff
@@ -22,6 +22,8 @@
   },
```

Comment:
Can you do a triage of this file to make sure we're using all of these dependencies and/or they're in the appropriate package directory?

Reply:
Yeah, this is an absolute necessity across all repos (as well as any Lerna sub-packages!)